core/api/v1/keytransparency_proto/keytransparency.proto

```diff
@@ -198,25 +194,20 @@ message UpdateEntryResponse {
   GetEntryResponse proof = 1;
 }
 
-// GetMutationsRequest contains the input parameters of the GetMutation APIs.
-message GetMutationsRequest {
+// GetEpochRequest identifies a particular epoch.
```

Comment:
Nit on the wording:  the parameters within GetEpochRequest are to identify the particular epoch, but the GetEpochRequest itself is to retrieve the mutations and related proofs

Could you say:  'GetEpochRequest identifies a particular epoch when retrieving the epoch's mutations and related proofs'?

Reply:
This is now `GetState`